pkg/controllers/securitypolicy/securitypolicy_controller.go

```diff
@@ -124,12 +123,7 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			return resultNormal, nil
 		}
 
-		if err := r.Service.OperateSecurityPolicy(obj); err != nil {
-			if errors.As(err, &util2.RestrictionError{}) {
-				log.Error(err, err.Error(), "securitypolicy", req.NamespacedName)
-				updateFail(r, &ctx, obj, &err)
-				return resultNormal, nil
-			}
```

Comment:
Why reverted this change?

Reply:
Refactored a long time ago, on vpc_dev is CreateOrUpdateSecurityPolicy,